cmd/kubeadm/app/util/apiclient/idempotency_test.go

```diff
@@ -148,35 +144,3 @@ func TestMutateConfigMap(t *testing.T) {
 		t.Fatalf("ConfigMap mutation was invalid, has: %q", cm.Data["key"])
 	}
 }
-
-func TestMutateConfigMapWithConflict(t *testing.T) {
```

Comment:
remove this test, since this test is specific for testing the retries of in `MutateConfigMap ()`